Source/EasyNetQ/Producer/Rpc.cs

```diff
@@ -171,13 +171,15 @@ protected void RegisterErrorHandling<TResponse>(Guid correlationId, Timer timer,
                             autoDelete: true);
 
                 var exchange = DeclareRpcExchange(conventions.RpcResponseExchangeNamingConvention(responseType));
-
-                advancedBus.Bind(exchange, queue, queue.Name);
+                if (exchange != Exchange.GetDefault())
+                {
+                    advancedBus.Bind(exchange, queue, queue.Name);
```

Comment:
it's not ok to bind to default exchange, you can't, and you get this exception:
```
The AMQP operation was interrupted: AMQP close-reason, initiated by Peer, code=403, text="ACCESS_REFUSED - operation not permitted on the default exchange", classId=40, methodId=10, cause=
```
so that piece of code is very important for this PR to work